### PR TITLE
Documentation retouch: cloud storages and mot data format

### PR DESCRIPTION
**Pull request by mdacoca**
- Added videos to Cloud Storages
- Updated wording in Cloud Storages
- Updated data format for MOT Import

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/advanced/formats/format-mot.md [hunk 1]
```diff
@@ -53,9 +53,10 @@ person
 Uploaded file: a zip archive of the structure above or:
 
 ```bash
-taskname.zip/
-├── labels.txt # optional, mandatory for non-official labels
```
Comment (Marishka17) [inline site/content/en/docs/manual/advanced/formats/format-mot.md:57]: I guess we don't need to delete the comment

**Review by Marishka17 — COMMENTED** [review]
We have `Attach storage to CVAT` for each provider, so I believe it would be better to insert video examples in each section and not create a new common section with examples at the end of the page. What do you think?

**Comment by Marishka17** [comment]
@mdacoca, Could you please recheck that the correct terms are used everywhere?

**Comment by mdacoca** [comment]
> @mdacoca, Could you please recheck that the correct terms are used everywhere?

updated

**Review by Marishka17 — CHANGES_REQUESTED** [review]

File: site/content/en/docs/manual/basics/attach-cloud-storage.md [hunk 2]
```diff
@@ -129,11 +133,13 @@ Fill in the following fields:
...
 
-### AWS manifest file
+### AWS S3 manifest file
+
+> **Note**: manifest file is optional.
 
 To prepare the manifest file, do the following:
 
-1. Go to [**AWS cli**](https://aws.amazon.com/cli/) and run
+1. Go to [**AWS S3 cli**](https://aws.amazon.com/cli/) and run
```
Comment (Marishka17) [inline site/content/en/docs/manual/basics/attach-cloud-storage.md:142]:
```suggestion
1. Go to [**AWS CLI**](https://aws.amazon.com/cli/) and run
```

**Review by Marishka17 — APPROVED** [review]

**Pull request merged**